src/utils/checkRequired.js

```diff
@@ -0,0 +1,7 @@
+const checkRequired = (data, message) => {
+  if (!data) {
+    return `${message} is required`;
+  }
+};
```

Comment:
Consider returning a default value (such as null or undefined) when the data is present. This will make it clear that the function always returns a value, which can be useful for consistency in error handling.